aws-glue/jobs/reporting-ratio_stats.py

```diff
@@ -1,4 +1,26 @@
-# Glue setup
+from assesspy import cod_ci as cod_ci
+from assesspy import boot_ci as boot_ci
+from assesspy import prb as prb
+from assesspy import cod as cod
+from assesspy import mki_met as mki_met
+from assesspy import mki as mki
+from assesspy import detect_chasing as detect_chasing
+from assesspy import prb_met as prb_met
+from assesspy import prd_met as prd_met
+from assesspy import cod_met as cod_met
+from assesspy import prd as prd
+import assesspy as ap
```

Comment:
**[Nitpick]** A couple notes here about imports:

1. The `as` keyword is only necessary if we want to create an alias for the package we're importing (e.g. `import assesspy as ap`). Since in these cases we're not changing the name of the packages we're importing, we don't need to include aliases.
2. If we want to import multiple symbols from one package, we can consolidate them into one `import` statement by separating the symbols with commas, e.g.:

```python
from assesspy import cod_ci, boot_ci, prb, ...
```

3. The symbol imports on lines 1-11 are duplicative with the `import assesspy as ap` call on line 12, and I can't tell if the import on line 12 is being used anywhere. We should either import these symbols from the package, or import the package directly, but not both.

Putting this all together, my suggestion would be:

```suggestion
from assesspy import (
    boot_ci,
    cod,
    cod_ci,
    cod_met,
    detect_chasing,
    mki,
    mki_met,
    prb,
    prb_met,
    prd
    prd_met,
)
```

Reply:
Reading the PR body again, it seems like this might be a result of difficulty refactoring to use the `assesspy` import:

> I've been having issues removing the ap. before the functions since it also renames the column names. Because of that they are imported separately. I have a structure in place and am working through it, but it's just time consuming running checks on it in AWS. I'll send a separate push if this is necessary to fix.

I don't quite understand what the trouble is, perhaps we could pair on it together?